angularSTEP/src/app/user.ts

```diff
@@ -0,0 +1,47 @@
+//User class that resembles the documents
+//in the Users collection on Firestore
+export class User {
+  username: string;
+  uid: string;
+  displayName: string;
+  email: string;
+  picUrl: string;
+  following: Array<string>;
+  time: number;
+  recipes: Array<string>;
+  wishlist: Array<string>;
+  shoppingList: object;
+  constructor(
+    username: string,
+    uid: string,
+    displayName: string,
+    email: string,
+    picUrl: string,
+    following: Array<string>,
+    time: number,
+    recipes: Array<string>,
+    wishlist: Array<string>,
+    shoppingList: object
+  ) {
```

Comment:
These could all be parameter properties.

Reply:
Oh wow, I didn't know you could do that in TypeScript so easily, thank you!